package/node/node/Makefile

```diff
@@ -35,24 +44,58 @@ define Package/npm
   URL:=https://npmjs.org/
 endef
 
+# From : https://github.com/tessel/openwrt-tessel/blob/bc3316cd4458a5359166daf5684e9906a9ef7851/package/node/node/Makefile
+# {
+CPU:=$(subst aarch64,arm64,$(subst x86_64,x64,$(subst i386,ia32,$(ARCH))))
+
+MAKE_VARS += \
+	DESTCPU=$(CPU)
+# }
+
 ifeq ($(ARCH),i386)
   DEST_CPU=ia32
 else
   DEST_CPU=$(ARCH)
 endif
 
 
-CONFIGURE_ARGS = --dest-cpu=$(DEST_CPU) --dest-os=linux \
-  --shared-openssl --shared-openssl-includes="$(STAGING_DIR)/usr/include" --shared-openssl-libpath="$(STAGING_DIR)/usr/lib" \
-  --shared-zlib --shared-zlib-includes="$(STAGING_DIR)/usr/include" --shared-zlib-libpath="$(STAGING_DIR)/usr/lib" \
-  --v8-options="--max_old_space_size=20 --initial_old_space_size=4 --max_semi_space_size=2 --max_executable_size=5" \
-  --without-snapshot
+CONFIGURE_ARGS = \
+	--dest-cpu=$(DEST_CPU) \
+	--dest-os=linux \
+  --without-snapshot \
+	--shared-zlib \
+	--shared-openssl \
+	--prefix=/usr \
+	--v8-options="--max_old_space_size=20 --initial_old_space_size=4 --max_semi_space_size=2 --max_executable_size=5 --optimize_for_size" \
```

Comment:
Searched but cannot learn all memory stuff in a short time. Added `--optimize_for_size` for good, without knowing if its valuable.
Also: https://github.com/nodejs/node/issues/7937